db/migrate/20220123100450_rename_deadline_fields_on_partner_groups.rb

```diff
@@ -0,0 +1,10 @@
+class RenameDeadlineFieldsOnPartnerGroups < ActiveRecord::Migration[6.1]
+  def change
+    safety_assured do
+      # Strong migrations complains that we should to this in multiple steps (new col, backfill, switch reads, delete old)
+      # I think it's fine as small scale.
```

Comment:
In my experience one of the issues with doing column renames is changing branches, either locally or on staging environments. Because the database state doesn't switch with the code branch, you end up breaking one or the other branch whenever it's deployed.

Will leave it up to @edwinthinks / @seanmarcia to see what they think as to the state of this app regarding the issue.